src/osd/PrimaryLogPG.cc

```diff
@@ -5862,7 +5879,10 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
 	  break;
 	}
 
-	if (op.extent.offset > cct->_conf->osd_max_object_size) {
+	if (op.extent.offset > osd_max_object_size) {
```

Comment:
Might as well swap this to use check_offset_and_length() while you're touching it.